recipes/morphsnakes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "morphsnakes" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4e946efc436bf309827453fe7487626004e1903ad3e65551265a86d0af1b63e9"
+
+build:
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - setuptools
+    - python
+    - scipy
```

Comment:
Is this needed in `host`?
```suggestion
```

Reply:
Unfortunately yes... the entire module is imported in setup.py to get the version number...